react-external/src/components/Molecules/Icons/Icons.tsx

```diff
@@ -0,0 +1,34 @@
+import { Stack, Typography } from "@mui/material";
+import Icon from "../../Atoms/Icon/Icon";
+
+function Icons() {
+  return (
+    <Stack direction="row" justifyContent="space-between">
+      <Stack direction="row" spacing={3} alignItems={"center"}>
+        <Icon
+          src={`${process.env.PUBLIC_URL}/asserts/Bike.svg`}
```

Comment:
should not pass the source directly either import it or get them from constants file